### DISCUSSION TITLE
how can i setup slurm configuration in espnet2?

### DISCUSSION THREAD

**Discussion by himanshucodz55** [espnet/espnet#4495] [Q&A]
Hi, i am running librispeech recipe on espnet2. i am running using slurm for distributed training. There is one script in conf/slurm.conf. I am new to this i do not know to set slurm.conf in espnet2. Could you pls provide sample slurm.conf if each machine has single gpu(i'm using 2 machine for distributed training), nCPU=12.
thanks in advance.

**Comment by sw005320** [comment]
I recommend you check https://kaldi-asr.org/doc/queue.html

**Reply by himanshucodz55** [reply]
Thank you @sw005320.
